hack/generator/pkg/astmodel/testdata/SetOptionalStringFromOptionalString-Direct.golden

```diff
@@ -0,0 +1,24 @@
+// Code generated by k8s-infra. DO NOT EDIT.
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+package vCurrent
+
+import "github.com/Azure/k8s-infra/hack/generated/_apis/Verification/vHub"
+
+type Person struct {
+	name *string `json:"name"`
+}
+
+// ConvertFrom populates our Person from the provided source Person
+func (person Person) ConvertFrom(source *vHub.Person) error {
+
+	// name
+	person.name = source.name
```

Comment:
This leaves the source and destination linked together, doesn't it? After this point the code could assign to `*source.name` and the value of `*person.name` would also be updated. Should there be a local variable to prevent this?

Reply:
Agreed. I'm splitting neither-optional from both-optional in a later PR, so I'll put the fix there.